test/unit/channels/facebook/test.channel.facebook.multiple_post.js

```diff
@@ -180,44 +212,423 @@ describe('Multi-post Unit Tests', () => {
       ]
     };
 
-    postResponse = {
-      duration: 250,
-      name: 'post',
+    const mock = nock(`${mockCloudFunctionsEndpoints.url}`)
+      .post(mockCloudFunctionsEndpoints.actionsEndpoint)
+      .reply(201, postResponse);
+
+    return mockMultiplePost.main(multiPostParams).then(
+      result => {
+        if (!mock.isDone()) {
+          nock.cleanAll();
+          assert(false, 'Mock server did not get called.');
+        }
+        assert.deepEqual(result, multiPostResponse);
+      },
+      error => {
+        assert(false, error);
+      }
+    );
+  });
+
+  it('validate multipost with arrayed message', () => {
+    const multiPostParams = {
+      recipient: {
+        id: 'xxx'
+      },
+      message: [
+        {
+          text: 'Here is your multi-modal response.'
+        },
+        {
+          attachment: {
+            type: 'image',
+            payload: {
+              url: 'https://xxx.com/xxx.png'
+            }
+          }
+        },
+        {
+          attachment: {
```

Comment:
Since there are under 11 options, the normalizer will convert this to a list of quick reply buttons.
The response JSON for Facebook will look as follows:
```json
{
	"text": "Choose your location",
	"quick_replies": [
		{
			"content_type": "text",
			"title": "Location 1",
			"payload": "Location 1"
		},
		{
			"content_type": "text",
			"title": "Location 2",
			"payload": "Location 2"
		},
		{
			"content_type": "text",
			"title": "Location 3",
			"payload": "Location 3"
		}
	]
}
```